[Direct PR] [release-18.0]: Add `sidecardb` known issue (#17096)
vitessio/vitess: changelog/18.0/18.0.0/summary.md

### Table of Contents

- **[Known Issues](#known-issues)**

>   - **[SideCarDB runs an ineffectual DDL](#sidecardb-issue)**

Reviewer: ```suggestion
  - **[VTTablet runs an unnecessary DDL](#sidecardb-issue)**
```

  - **[Durability Policies](#durability-policies)**
    - [New Durability Policies](#new-durability-policies)

## <a id="known-issues"/>Known Issues

### <a id="sidecardb-issue"/>SideCarDB runs an ineffectual DDL

> In this release, with MySQL 8.0 whenever SideCarDB runs, it finds

Reviewer: ```suggestion
In this release, with MySQL 8.0 whenever VTTablet's tabletserver goes through initialization, it finds
```

> a diff in the schemacopy table and tries to apply a DDL to fix it. The DDL

Reviewer: ```suggestion
a diff in its internal schemacopy table and tries to apply a DDL to fix it. The DDL
```

Reviewer: Can you back quote the table name?
